app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ProfilesController < ApplicationController
+  def show
+    @user = current_user
+  end
+
+  def update
+    @user = current_user
```

Comment:
why do you need @user? can you use current_user???

Reply:
done